src/pages/overview/pdf-services-api/howtos/protect-pdf.md

```diff
@@ -34,36 +34,40 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
-                ExecutionContext executionContext = ExecutionContext.create(credentials);
-     
-                // Build ProtectPDF options by setting a User Password and Encryption
-                // Algorithm (used for encrypting the PDF file).
-                ProtectPDFOptions protectPDFOptions = ProtectPDFOptions.passwordProtectOptionsBuilder()
-                        .setUserPassword("encryptPassword")
+                // Initial setup, create credentials instance
+                Credentials credentials = new ServicePrincipalCredentials(
+                        System.getenv("PDF_SERVICES_CLIENT_ID"),
+                        System.getenv("PDF_SERVICES_CLIENT_SECRET"));
+    
+                // Creates a PDF Services instance
+                PDFServices pdfServices = new PDFServices(credentials);
+    
+                // Creates an asset from source file and upload
+                InputStream inputStream = Files.newInputStream(new File("src/main/resources/protectPDFInput.pdf").toPath());
+                Asset asset = pdfServices.upload(inputStream, PDFServicesMediaType.PDF.getMediaType());
+    
+                // Create parameters for the job
+                ProtectPDFParams protectPDFParams = ProtectPDFParams.passwordProtectOptionsBuilder()
+                        .setUserPassword("password")
                         .setEncryptionAlgorithm(EncryptionAlgorithm.AES_256)
                         .build();
-     
-                // Create a new operation instance.
-                ProtectPDFOperation protectPDFOperation = ProtectPDFOperation.createNew(protectPDFOptions);
-     
-                // Set operation input from a source file.
-                FileRef source = FileRef.createFromLocalFile("src/main/resources/protectPDFInput.pdf");
-                protectPDFOperation.setInput(source);
-     
-                // Execute the operation
-                FileRef result = protectPDFOperation.execute(executionContext);
-     
-                // Save the result at the specified location
-                result.saveAs("output/protectPDFOutput.pdf");
-     
-            } catch (ServiceApiException | IOException | SdkException | ServiceUsageException ex) {
+    
+                // Creates a new job instance
+                ProtectPDFJob protectPDFJob = new ProtectPDFJob(asset, protectPDFParams);
+    
+                // Submits the job and gets the job result
+                String location = pdfServices.submit(protectPDFJob);
+                PDFServicesResponse<ProtectPDFResult> pdfServicesResponse = pdfServices.getJobResult(location, ProtectPDFResult.class);
+    
+                // Get content from the resulting asset(s)
+                Asset resultAsset = pdfServicesResponse.getResult().getAsset();
+                StreamAsset streamAsset = pdfServices.getContent(resultAsset);
+    
+                // Creates an output stream and copy stream asset's content to it
```

Comment:
*copies if  creates being used

Reply:
As discussed, IG this is from already reviewed samples.